app/src/client/endpoints.ts

```diff
@@ -124,6 +124,35 @@ export class GetCollectionStatisticsResponse extends ResponseData {
     );
   }
 }
+export class GetTokenRecentSalesRequest extends RequestData {
+  readonly offset: number;
+  constructor(offset?: number) {
+    super();
+    this.offset = offset || 0;
+  }
+  public toObject = (): Record<string, unknown> => {
```

Comment:
also, you need newlines between things in classes. when you write code please look at the code around it and match!